appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java

```diff
@@ -83,56 +88,83 @@ public void onValueChange(ValueChangeEvent<Boolean> event) {
     });
     phoneScreen.add(checkboxShowHiddenComponents);
 
-    checkboxPhoneTablet = new CheckBox(MESSAGES.previewPhoneSize()) {
+    listboxPhoneTablet = new ListBox() {
       @Override
       protected void onLoad() {
-        // onLoad is called immediately after a widget becomes attached to the browser's document.
```

Comment:
You might not want to remove the comment.

Reply:
Done